Fix clientId for new tab

zumkorn opened the pull request:
This issue https://github.com/logux/logux/issues/74

ai commented:
Can you add a small test to avoid regressions? For instance, by checking `client` key [here](https://github.com/logux/client/blob/master/cross-tab-client/index.test.ts#L618).

zumkorn commented:
I added a test, but one test broke.
https://github.com/logux/client/blob/master/client/index.test.ts#L683
 I corrected it, but I'm not sure if I did it right.

ai commented:
LGTM. I will accept and release it today

ai commented:
Thanks. Released in 0.9.3.

ai commented:
BTW, do you have a Twitter account to mention you in the release tweet?

zumkorn commented:
> BTW, do you have a Twitter account to mention you in the release tweet?

https://twitter.com/zumkorn
